website/next.config.js

```diff
@@ -3,9 +3,10 @@ const path = require('path')
 
 module.exports = withHashicorp({
   defaultLayout: true,
-  transpileModules: ['is-absolute-url', '@hashicorp/react-mega-nav'],
+  transpileModules: ['is-absolute-url', '@hashicorp/react-.*'],
   mdx: { resolveIncludes: path.join(__dirname, 'pages') },
 })({
+  svgo: { plugins: [{ removeViewBox: false }] },
```

Comment:
Definitely a good candidate to add to nextjs-scripts - would you mind putting in a PR for this whenever you have a moment?